README.md

```diff
@@ -41,6 +41,11 @@ Visualizes a collection following the STAC-based collection description.
 - `version` (string): openEO version (defaults to `null`, which tries to auto-detect the version).
 - `collectionData` (object): A single STAC-based collection object as defined by the openEO API.
 - `initiallyCollapsed` (boolean): Allow collapsing/expanding the details and collapse the details by default (defaults to `false`).
+- `mapOptions` (object): For fine-tuning the behaviour of the map that displays the collection's spatial extent. Entirely optional. Possible keys and their default values:
+  - `height` (string) -- `"300px"`: Height of the map container div
```

Comment:
In all other docs we wrote something like "(defaults to `false`)" or similar instead of the "-- value". Please align that. One way or the other...

Reply:
I saw that but thought because it's a sublist a custom pattern would be fine. But I changed it anyway.